build/travis/debug-pods.sh

```diff
@@ -9,10 +9,8 @@ do
 	for SS in "${SSS[@]}"; do
 	  if [ -n "${SS}" ]; then
 	    echo "${SS}" "is ready for logs"
-	    kubectl get pods -o yaml >> debug.log
```

Comment:
@ryanemerson I think we can remove this script as you've implemented more intelligent solution for Pod logs, and this script was a quick-fix workaround

Reply:
I didn't know if the Infinispan and Statefulset yaml might be useful? I suppose I can add this to the test method as well.

Reply:
Sometimes we need SS status to identify why Pods can't be created and Infinispan CR status in other cases, but we can trace them inside code also as your solution for Pods